README.md

```diff
@@ -1,7 +1,212 @@
-# src-mq 
+# src-mq
 
-> Breakpoint media queries in JS.
+🤓 Lucid breakpoints for JavaScript.
 
-[![Build Status](https://travis-ci.org/src-mq/src-mq.svg?branch=master)](https://travis-ci.org/src-mq/src-mq) 
+[![Build Status](https://travis-ci.org/src-mq/src-mq.svg?branch=master)](https://travis-ci.org/src-mq/src-mq)
 [![codecov](https://codecov.io/gh/src-mq/src-mq/branch/master/graph/badge.svg)](https://codecov.io/gh/src-mq/src-mq)
+![npm type definitions](https://img.shields.io/npm/types/src-mq)
+![npm bundle size](https://img.shields.io/bundlephobia/minzip/src-mq)
 
+### Example
+
+```js
+import { from, until } from 'src-mq'
+
+const styles = {
+
+	[from.small()]: { ... },
+
+	[until.large()]: { ... },
+
+	[from.small.until.large()]: { ... },
+
+	[from.small.until.large.for.screen()]: { ... },
+
+}
+```
+
+## API
+
+### `from<breakpoint>`
+
+Type: `function`
+
+Returns a media query scoped to:
+
+- a minimum width of **breakpoint**
+
+```scss
+/* {[from.small()]: { ... }} */
+
+@media all and (min-width: 30em) { ... }
+```
+
+### `from<breakpoint>.for[screen, print, speech]`
+
+Type: `function`
+
+Returns a media query scoped to:
+
+- a minimum width of **breakpoint**
+- the specified media type
+
+```scss
+/* {[from.small.for.screen()]: { ... }} */
+
+@media screen and (min-width: 30em) { ... }
+```
+
+### `until<breakpoint>`
+
+Type: `function`
+
+Returns a media query scoped to:
+
+- a maximum width of **breakpoint** − 1px
+
+```scss
+/* {[until.large()]: { ... }} */
+
+@media all and (max-width: 61.1875em) { ... }
+```
+
+### `until<breakpoint>.for[screen, print, speech]`
+
+Type: `function`
+
+Returns a media query scoped to:
+
+- a maximum width of **breakpoint** − 1px
+- the specified media type
+
+```scss
+/* {[until.large.for.screen()]: { ... }} */
+
+@media screen and (max-width: 61.1875em) { ... }
+```
+
+### `from<fromBreakpoint>.until<untilBreakpoint>`
+
+Type: `function`
+
+Returns a media query scoped to:
+
+- a minimum width of **fromBreakpoint**
+- a maximum width of **untilBreakpoint** − 1px
+
+```scss
+/* {[from.small.until.large()]: { ... }} */
+
+@media all and (min-width: 30em) and (max-width: 61.1875em) { ... }
+```
+
+### `from<fromBreakpoint>.until<untilBreakpoint>.for[screen, print, speech]`
+
+Type: `function`
+
+Returns a media query scoped to:
+
+- a minimum width of **fromBreakpoint**
+- a maximum width of **untilBreakpoint** − 1px
+- the specified media type
+
+```scss
+/* {[from.small.until.large.for.screen()]: { ... }} */
+
+@media screen and (min-width: 30em) and (max-width: 61.1875em) { ... }
+```
+
+## Defaults
+
+_src-mq_ provides a reasonable set of defaults:
+
+#### Media type
+
+`all`
+
+#### Breakpoints
+
+- `xxSmall` (320 pixels)
+- `xSmall` (375 pixels)
+- `small` (480 pixels)
+- `medium` (740 pixels)
+- `large` (980 pixels)
+- `xLarge` (1140 pixels)
+- `xxLarge` (1300 pixels)
+
+_Breakpoint values are output to CSS as ems, assuming 1em = 16px._
```

Comment:
I guess this should be made configurable in the long term?

Reply:
ems vs px etc?